ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/telephoneinput/v1/telephoneinput/telephoneinput.html

```diff
@@ -32,7 +32,7 @@
     <div data-sly-call="${label.label @componentId=telText.id, labelValue=telText.label.value, labelVisible=telText.label.visible, bemBlock='cmp-adaptiveform-telephoneinput'}" data-sly-unwrap></div>
     <input title="${telText.tooltipVisible ? '' : telText.tooltipText}"
            class="cmp-adaptiveform-telephoneinput__widget"
-           aria-label="${telText.label.visible && telText.label.value}"
+           aria-label="${telText.label.value}"
```

Comment:
why aria-label is required when it can be taken care by the `label for `itself.

Reply:
![Uploading Screenshot 2023-06-29 at 4.35.42 PM.png…]()

Reply:
@nit23uec 
<img width="658" alt="Screenshot 2023-06-29 at 4 35 42 PM" src="https://github.com/adobe/aem-core-forms-components/assets/33446537/904e57a7-78e0-4031-8a09-9818ba865722">